code/modules/mob/living/carbon/martian/martian.dm

```diff
@@ -115,6 +115,7 @@
 		update_inv_head(redraw_mob)
 
 	W.layer = 20
+	W.plane = PLANE_HUD
```

Comment:
Why is this stuff not a proc then people would stop forgetting to do it all aaaaa